### DISCUSSION TITLE
Regarding proof of your academic status for github student developer pack

### DISCUSSION THREAD

**Discussion by sushi32** [community/community#42018] [GitHub Education]
I am a master in computer science student at the University at Buffalo (SUNY Buffalo), in order to prove my academic status for the GitHub student developer pack, in website it is specified to upload a student ID card with an expiration date, but there is no valid until date on our student ID card.
Also since currently I am in my first semester hence I do not have any transcripts yet.
Below are the documents I could submit,
1.	Student ID card - but no expiration date.
2.	I have an I20 with course duration (As I am an international student)
3.	I can get the class schedule for this semester and next semester with my Name on the document.

When I merge these documents and submit them for proof, it throws me errors as below:

Your application cannot be reviewed until you fix the following:
 > You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
> Your image should be one of the following: Dated school ID

![image](https://user-images.githubusercontent.com/39491839/208272542-fd1be7e0-7c19-48da-ad2d-cc1c791826da.png)

Did anyone else face the same issue?

Please guide me on how I should proceed with the application.

**Comment by engirugger42** [comment]
I wish I had an answer, but I'm currently facing a similar issue. Online masters in cyber security from Southern New Hampshire University (SNHU). Can't even get to the location part because of the same issues with documentation size and dates. Hopefully a response here will keep your topic in view of others.

**Comment by samus-aran** [comment]
Hi @sushi32 👋 

We're happy you're here. Let's see how we can assist. Although we do prefer Dated Student ID we understand and accommodate that some schools simply don't issue these but Student ID is still their most preferred form of evidence. So please select the most appropriate evidence type in the dropdown which sounds like ~Dated~ Student ID. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have a Student ID (for example) do try and acquire evidence that is positioned higher rather than lower in that option list.
When being a remote student we recommend that the evidence states this. However if you had the opportunity to apply whilst on campus that would also be beneficial to affirming your student status.

Hope this all helps. Let us know how you get on. 👍

**Reply by engirugger42** [reply]
I'm sorry, but I'm a bit confused by this suggestion. I tried the application again, but this time from my phone, I had to do this because the upload a file option was missing altogether this time. I took a picture of my tuition payment and student portal, but still received location errors and suggestions that my request data was not sufficient, canceled the application, and now it says I have an outstanding request despite my canceling it. 

Is there anybody we can reach out to see if a request is actually outstanding and fully filled out?

**Comment by Ca22io** [comment]
Estou passando pela mesma situação de estar dando que estou longe do campus, sendo que o estou no mesmo todos os dias.
Já tentei mudar a localização ou até mesmo fazer a solicitação no mesmo.
